Hakawai/Mentions/HKWMentionsPlugin.m

```diff
@@ -1680,11 +1722,11 @@ - (void)textViewDidChangeSelectionV2:(UITextView *)textView {
     // Find a mentions query from the last control char if there is one
     NSString *query = [self mentionsQuery:textView.text location:range.location];
     if (query) {
-        // query it
-        [self beginMentionsCreationWithString:query
+        // first character is control character,rest of string is query
+        [self beginMentionsCreationWithString:[query substringFromIndex:1]
```

Comment:
what if it's empty query?

Reply:
then it will return an empty string https://developer.apple.com/documentation/foundation/nsstring/1414368-substringfromindex?language=objc